### ISSUE TITLE
IMPORTANT: unclear legal status

### ISSUE THREAD

**Issue by notpushkin** [CorruptionHades/DreamBeardCompiler#1]
Technically this project is governed by the DreamBerd Foundation, as it [doesn't include the word DreamBerd in title](https://github.com/TodePond/DreamBerd#ownership).

However, **DreamBeard looks a lot like DreamBerd**, so people might get confused and think it's your own project and is not affiliated with DreamBerd. It also **might have violated trademark law** if DreamBerd was a registred trademark, which to the best of my knowledge it isn't.

In conclusion, I suggest not doing anything and just leaving the project in this uncertain state. This way you'll have plausible deniability if things go to court. (This is not legal advice. No lawyer-client, solicitor-client or attorney-client relationship shall be created through the use of this information. I'm not even an attorney.)

**Comment by CorruptionHades** [comment]
I renamed the repo

**Comment by TodePond** [comment]
thank you
